lua/entities/gmod_tardis_interior/modules/sh_portals.lua

```diff
@@ -44,7 +44,7 @@ else
 		end
 		if dont then
 			return false, black
-		elseif (not (self.DoorOpen and self:DoorOpen(true))) and portal==self.portals.interior then
+		elseif (not (self.DoorOpen and self:DoorOpen(false))) and portal==self.portals.interior then
```

Comment:
What was the reason for this change?

Reply:
Interior door closes slower
This prevents the interior portal from disappearing before it's closed

Reply:
Okay, hopefully this has no negative side effects but I suppose testing will reveal that, just got to be careful messing with core rendering code